### DISCUSSION TITLE
NextUI component props are not working when deploying to Vercel

### DISCUSSION THREAD

**Discussion by DriesVanHool** [heroui-inc/heroui#2194] [Help] [answered]
Hi,

I'm using Vite, and for some reason the only styling that is not working when deploying my app to Vercel are the onces I declare with the NextUI props like <Button variant="bordered"></Button>

`import { defineConfig } from 'vite'
import react from '@vitejs/plugin-react-swc'

Vite config

```
export default defineConfig({
  plugins: [react()],
})
```

Tailwind config
import {nextui} from "@nextui-org/react";

```
/** @type {import('tailwindcss').Config} */
export default {
  content: [
    "./index.html",
    "./src/**/*.{js,ts,jsx,tsx}",
    "./node_modules/@nextui-org/theme/dist/**/*.{js,ts,jsx,tsx}"
  ],
  theme: {
    extend: {},
  },
  darkMode: "class",
  plugins: [nextui()]
}
```

Post css config
```
export default {
  plugins: {
    tailwindcss: {},
    autoprefixer: {},
  },
}
```

Locally everything works fine

**Comment by DriesVanHool** [comment, marked as answer]
I needed to add the content of my npmrc file in my vercel env variables with key **NPM_RC** to add public-hoist-pattern[]=*@nextui-org/*

**Reply by guugg** [reply]
I added `NPM_RC` and it didn't work, I used pnpm

**Reply by guugg** [reply]
I deleted the vercel entry, recreated it, and added RC,_RC, NPM_RC, all pointing to public-hoist-pattern []=*@ next-org/*. He did it. The pnpm global configuration used by my native Windows 11 is the RC file.
